server/src/Packages/Paypal/Services/PayPalClient.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace  Gernzy\Server\Packages\Paypal\Services;
+
+use Gernzy\Server\Exceptions\GernzyException;
+use PayPalCheckoutSdk\Core\PayPalHttpClient;
+use PayPalCheckoutSdk\Core\ProductionEnvironment;
+use PayPalCheckoutSdk\Core\SandboxEnvironment;
+
+ini_set('error_reporting', E_ALL); // or error_reporting(E_ALL);
```

Comment:
Maybe for debugging scenarios we can wrap this in a universal config conditional? Like `$debug = config('app.debug');`

Reply:
completed